package.json

```diff
@@ -59,12 +62,11 @@
     "start": "react-scripts start",
     "start-untyped": "TSC_COMPILE_ON_ERROR=\"true\" react-scripts start",
     "build": "react-scripts build",
-    "relay": "npx relay-compiler --verbose --watchman false --src ./src --schema schema.graphql --language typescript",
-    "sync-schema": "get-graphql-schema https://api.cirrus-ci.com/schema.json | sed -e 's/Long/Int/g' > schema.graphql",
```

Comment:
Let's keep `relay` and `sync-schema` to have a way to call just them. Plus `bootstrap` can just call them instead of inlining too many commands in one `bootstrap`

Reply:
Okay.  Just so you know, only my docs PRs are ready, this one needs a few more minutes

Reply:
Exactly what I meant. Please open PRs only once they are ready for a review in the future. ☝️